verification/src/tests/parse.rs

```diff
@@ -0,0 +1,26 @@
+use std::fmt::Debug;
+
+use serde::{de::DeserializeOwned, Serialize};
+use serde_json::Value;
+
+pub fn test_deserialize_ok<T>(tests: Vec<(&str, T)>)
+where
+    T: Debug + PartialEq + DeserializeOwned,
+{
+    for (s, value) in tests {
+        let v: T = serde_json::from_str(s).unwrap();
+        assert_eq!(v, value);
+    }
+}
+
+pub fn test_serialize_json_ok<T>(tests: Vec<(T, Value)>)
+where
+    T: Debug + PartialEq + Serialize,
```

Comment:
```suggestion
    T: Serialize,
```
Seems that the code compiles without `Debug` and `PartialEq` traits